src/components/Transaction/Transactions.js

```diff
@@ -9,26 +9,31 @@ export default function Transactions() {
 	useStore(state => state.transactions);
 	const {accountID} = useParams();
 	const getAccountWithData = useStore(state => state.getAccountWithData);
+	const getTransactionWithData = useStore(state => state.getTransactionWithData);
 	const account = getAccountWithData(accountID);
 	const navigate = useNavigate();
 	const deleteTransaction = useStore(state => state.deleteTransaction);
 
 	return (
 		<>
-			{account.Transactions.map(({id, date, change, note}) => (
-				<Transaction
-					key={id}
-					date={date}
-					change={change}
-					note={note}
-					onEdit={() => {
-						navigate('/transaction/edit/' + accountID + '/' + id);
-					}}
-					deleteTransaction={() => {
-						deleteTransaction(id);
-					}}
-				/>
-			))}
+			{account.Transactions.map(({id, date, change, note}) => {
```

Comment:
Is this a mistake, or is there a specific reason  why you name the transactions with a capital `T` (cf. `useStore`). This is a bit confusing not only because your component is also called `Transactions`.

Reply:
It´t no mistake. We set a new attribute in our getAccountWithData method in useStore. The captial `T` was recommend by Gregor. If I remember correctly it´s to show that the data behind the key Transaction is selected and filtert from the the "Transaction Database".

Reply:
Ahhh... I think got it 😉